app/models/joke.rb

```diff
@@ -1,4 +1,12 @@
 class Joke < ApplicationRecord
   validates :question, presence: true
   validates :answer, presence: true
+
+  def full
+    question + " " + answer
+  end
+
+  def self.random
+    order("RANDOM()").sample
+  end
```

Comment:
i think i like it on `Joke` more than user. then we'd also update the rake task to record a new `JokeUser` record in the db? is that what you're thinking?